src/editors/data/constants/requests.js

```diff
@@ -0,0 +1,20 @@
+import { StrictDict } from '../../utils';
+
+export const ReqeustKeys = StrictDict({
```

Comment:
Just to note you can get rid of this here-- unless a `request` is different than a `Reqeust` ;)

Reply:
Yeah, I was pointing out that you have a misspelled duplicate in this file lol